mut/stage.py

```diff
@@ -180,9 +186,12 @@ def print(self) -> None:
 
 class ChangeSet:
     """Stores a list of S3 bucket operations."""
-    def __init__(self, verbose: bool) -> None:
+    def __init__(self, verbose: bool, deployed_url_prefix: str, return_json: bool) -> None:
         self.verbose = verbose
         self.suspicious_files = []  # type: List[str]
+        self.full_deploy_urls = []  # type: List[str]
+        self.deployed_url_prefix = deployed_url_prefix
+        self.return_json = return_json
```

Comment:
This should probably be a parameter to `ChangeSet.print()`, not an instance attribute.

But we should probably actually get away from scattering `print()` calls all over the place, and instead have a global singleton logger that we can swap out as appropriate in `main()`.